test/DynamoCoreTests/Logging/DynamoLoggerTest.cs

```diff
@@ -45,27 +63,9 @@ public void test_dynamologger_LogEventArgs()
         [Category("UnitTests")]
         public void test_dynamologger_logger()
         {
-            //Arrange
-            //This has been done in the base class but we need to do it again because the pathManager is private
-            var config = CreateStartConfiguration(dynamoSettings);
-            pathManager = new PathManager(new PathManagerParams
-            {
-                CorePath = config.DynamoCorePath,
-                HostPath = config.DynamoHostPath,
-                PathResolver = config.PathResolver
-            });
```

Comment:
Just to confirm, these params were not needed, right?

Reply:
yeah, these were null values.